apps/meteor/client/views/room/MessageList/components/MessageContentBody.tsx

```diff
@@ -33,100 +11,7 @@ type MessageContentBodyProps = {
 const MessageContentBody = ({ message }: MessageContentBodyProps): ReactElement => {
```

Comment:
Since you have moved most of this logic to another component, couldn't we just parse the message outside this and directly use MessageMarkup?

Reply:
This way we can remove this component.